js/adapt-contrib-branching.js

```diff
@@ -23,12 +25,23 @@ class Branching extends Backbone.Controller {
     const config = Adapt.course.get('_branching');
     if (!config || !config._isEnabled) return;
     // Wait for all other app:dataReady handlers to finish
+    if (this._isAwaitingDataReady) return;
+    this._isAwaitingDataReady = true;
     await data.whenReady();
+    this.warnForSpoorMisconfiguration();
+    this._isAwaitingDataReady = false;
     this.setupBranchingModels();
     this.setupEventListeners();
     Adapt.trigger('branching:dataReady');
   }
 
+  warnForSpoorMisconfiguration() {
+    const config = Adapt.config.get('_spoor');
+    const isMisConfigured = (config?._isEnabled && config?._tracking?._shouldStoreAttempts === false);
+    if (!isMisConfigured) return;
```

Comment:
```suggestion
    if (!isMisconfigured) return;
```

(Minor typo)

Reply:
You can do multi-line suggestions, if you use shift to select and then click the comment plus at the bottom of the selection.